### ISSUE TITLE
Performance of the standalone model

### ISSUE THREAD

**Issue by vaishnavh** [tsitsimis/neural-network-distillation#1]
Hey! I was just browsing around for some toy code for distillation and found yours. It's very useful!

If I'm not wrong, I think that the standalone small model's loss should have the `from_logits` set to be `False` because the mode's output layer already has a softmax?  When I changed this, I observed that it achieves the same error as the distilled student model.
